molecule/packages-dynamic/molecule.yml

```diff
@@ -0,0 +1,54 @@
+---
+dependency:
+  name: galaxy
+  role-file: requirements.yml
+driver:
+  name: docker
+lint: |
+    yamllint .
+    ansible-lint
+# This is ignored by molecule, but can be reused in yaml
+.platform_base: &platform_base
+  privileged: False
+  volumes:
+    - /sys/fs/cgroup:/sys/fs/cgroup:ro
+  tmpfs:
+    # Fixes issues accessing .so files under /tmp during pip builds.
+    # https://github.com/docker/compose/issues/1339
+    - /tmp:exec,mode=1777
+    - /run
+    - /run/lock
+  capabilities:
+    - NET_ADMIN
+    - NET_RAW
+platforms:
+  - <<: *platform_base
+    name: centos-7
+    image: centos:7
+    command: /sbin/init
+  - <<: *platform_base
+    name: centos-8
+    image: centos:8
+    command: /sbin/init
```

Comment:
why not test fedora?

Reply:
No repo available to test with; katello's "pulpcore" repo is only el7 & el8. And I never finished getting pulpcore included in Fedora itself.